torch/serialization.py

```diff
@@ -391,8 +391,8 @@ def load(f, map_location=None, pickle_module=pickle, **pickle_load_args):
         ``UnicodeDecodeError: 'ascii' codec can't decode byte 0x...``. This is
         caused by the difference of handling in byte strings in Python2 and
         Python 3. You may use extra :attr:`encoding` keyword argument to specify how
-        these objects should be loaded, e.g., :attr:`encoding='latin1'` decodes them
-        to strings using ``latin1`` encoding, and :attr:`encoding='bytes'` keeps them
+        these objects should be loaded, e.g., :attr:`encoding='utf-8'` decodes them
+        to strings using ``utf-8`` encoding, and :attr:`encoding='bytes'` keeps them
         as byte arrays which can be decoded later with ``byte_array.decode(...)``.
```

Comment:
Probably want to mention we default to utf-8 here?

Reply:
Yes, good idea. Will add in a bit.